src/app/Dashboard/RecentResourceItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+// import { Link } from 'react-router-dom';
+
+
+export default function RecentResourceItem(props) {
+	let TopicItem;
+
+	/**
+	 * This function passes in a topics array and item object from a higher level component
+	 * a forEach loop is called on the topics array to find the name of a resources parent topic
+	 * Once found it is assinged to the TopicItem variable
+	 * @param {{topics:array}} topics 
+	 * @param {{item:object}} item 
+	 */
+
+	const getTopicTitle = (topics, item) => {
```

Comment:
With the new seed data model, we can get rid of this extraneous function.

Reply:
removed function because it is no longer needed